treeherder/client/thclient/client.py

```diff
@@ -701,6 +711,16 @@ def _post_json(self, project, endpoint, oauth_key, oauth_secret, jsondata,
                              timeout=timeout)
         resp.raise_for_status()
 
+    def get_revision_jobs(self, repo_name, revision):
```

Comment:
As I said on irc, I think we should try to make this a thin wrapper around treeherder's rest api. So for now, let's make this method take a revision_id as a parameter, and then have it call the jobs API directly